lib/packages/shared-types/opensearch/main/transforms/issue-rai.ts

```diff
@@ -3,7 +3,7 @@ import { raiIssueSchema } from "../../..";
 export const transform = (id: string) => {
   return raiIssueSchema.transform((data) => ({
     id,
-    makoChangedDate: !!data.timestamp
+    makoChangedDate: data.timestamp
```

Comment:
These concern me only because I don't know if I can trust the type here enough to remove it. If we are able to confirm that the code sufficiently validates the type is what we say it is I am okay with this